src/components/shift-schedule/AdministratorMenu.tsx

```diff
@@ -0,0 +1,58 @@
+'use client';
+
+import { Button } from '@/components/ui/Button';
+import {
+  Collapsible,
+  CollapsibleContent,
+  CollapsibleTrigger,
+} from '@/components/ui/Collapsible';
+import { ChevronDownIcon, ChevronUpIcon, Trash2Icon } from 'lucide-react';
+import { useState } from 'react';
+
+type AdministratorMenuProps = {
+  messages: {
+    administratorMenu: string;
+    clearShiftSchedule: string;
+  };
+};
+
+function AdministratorMenu({ messages }: AdministratorMenuProps) {
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <Collapsible
+      open={isOpen}
+      onOpenChange={setIsOpen}
+      className='mx-auto xs:mx-8 my-8 rounded border p-3'
+    >
+      <section className='mx-1 flex justify-between'>
```

Comment:
I havent look at where this is used in the HTML, but I think using a section tag here is semantically wrong and we should use a div. A section is often used to indicate different parts of an article when using that